marklogic-data-hub/build.gradle

```diff
@@ -57,9 +57,9 @@ dependencies {
         exclude group: 'com.marklogic', module: 'marklogic-client-api'
     }
 
-    compile group: 'org.springframework.boot', name: 'spring-boot', version: '2.1.3.RELEASE'
-    compile group: 'org.springframework.integration', name: 'spring-integration-http', version: '5.1.3.RELEASE'
-    compile group: 'org.springframework.boot', name: 'spring-boot-autoconfigure', version: '2.1.3.RELEASE'
+    compile group: 'org.springframework.boot', name: 'spring-boot', version: '2.1.15.RELEASE'
```

Comment:
Just an FYI (no action needed) - I soon want to get rid of Spring Boot from the core library. Including it causes issues for applications that want to depend on DHF core as a dependency. Ideally, we only depend on the Spring core stuff in DHF core, and then QS and HC depend on Spring Boot, since they're Spring Boot applications. Maybe something for 5.4.